tests/stdlib_server_tests.nim

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2021 Status Research & Development GmbH
+# Licensed and distributed under either of
+#   * MIT license: http://opensource.org/licenses/MIT
+#   * Apache License, Version 2.0: http://www.apache.org/licenses/LICENSE-2.0
+# at your option. This file may not be copied, modified, or distributed except according to those terms.
+
+import ../metrics, ../metrics/stdlib_httpserver,
+  os, httpclient
+
+startMetricsHttpServer()
+sleep(1000)
+when defined(metrics):
```

Comment:
Why this not using `curl` too?

Reply:
Because I wrote it first, and was able to find an HTTP client in the library's docs.